.github/workflows/publish-autoinstrumentation-apache-httpd.yaml

```
@@ -19,12 +19,14 @@ concurrency:
 
 permissions:
   contents: read
-  packages: write
-  attestations: write
-  id-token: write
 
 jobs:
   publish:
+    permissions:
+      contents: read
+      packages: write # required for pushing Docker images to GHCR
+      attestations: write # required for generating build attestations
+      id-token: write # required for OIDC token access
```

make this change throughout all the files
```suggestion
    permissions:
      packages: write
      attestations: write
      id-token: write
```